### PR TITLE
Fix allowance for padding after lookupswitch bytecode

### LINKED ISSUE #7349: DDR command !bytecodes fails to dump the bytecodes of a method.
System core file is available in our S drive.

The above system core file was generated from Java 8 SR 5 FP40, and I am using the jdmpview from Java 8 SR 5 FP40:
```
> !bytecodes 0x00000000017D76F8
  Name: processIncomingContext
  Signature: (Lcom/ibm/CORBA/iiop/ORB;Lcom/ibm/CORBA/channel/giop/GIOPMessageContext;Lcom/ibm/CORBA/channel/giop/GIOPConnectionContext;Lcom/ibm/wsspi/iiop/channel/ConnectionStateElement;)V
  Access Flags (10270001): public
  Max Stack: 6
  Thrown Exceptions (1):
java/lang/Exception  Argument Count: 5
  Temp Count: 10

    0 getstatic 34 com/ibm/ws390/channel/ziop/ORBServiceContextFilter.tc Lcom/ibm/ejs/ras/TraceComponent;
    3 invokevirtual 35 com/ibm/ejs/ras/TraceComponent.isEntryEnabled()Z
    6 ifeq 29
    9 getstatic 34 com/ibm/ws390/channel/ziop/ORBServiceContextFilter.tc Lcom/ibm/ejs/ras/TraceComponent;
   12 ldc 1 (java.lang.String) processIncomingContext
   14 iconst2
   15 anewarray 36 java/lang/Object
..........................................
  107 iload 13
  109 lookupswitch pairs 1280
        default      96109
            256      29549
           1536      46189
           4352      79469
     1112343808      13238
     1112347136      62902
       15150080Problem running command:
com.ibm.j9ddr.CorruptDataException: com.ibm.j9ddr.InvalidDataTypeException: UDATA contains value larger than Integer.MAX_VALUE
>
```
Let me know if you need the .class file.  The corresponding class is com/ibm/ws390/channel/ziop/ORBServiceContextFilter.  Dumping the bytecodes from javap worked:

```
       101: iconst_0
       102: invokevirtual #15                 // Method java/nio/ByteBuffer.getInt:(I)I
       105: istore        13
       107: iload         13
       109: lookupswitch  { // 5
                       1: 224
                       6: 289
                      17: 419
              1229081861: 160
              1229081874: 354
                 default: 484
            }
       160: aload_2
       161: invokeinterface #16,  1           // InterfaceMethod com/ibm/CORBA/channel/giop/GIOPMessageContext.getType:()I
       166: ifne          484
       169: aload         11
       171: iload         12
       173: aaload
       174: bipush        8
       176: invokevirtual #17                 // Method java/nio/ByteBuffer.position:(I)Ljava/nio/Buffer;
       179: pop
```

### PR DESCRIPTION
**Pull request by keithc-ca**
* extract and reuse behavior from tableswitch bytecode
* reduce cost of setting j9bcutil_dumpBytecodes in the environment

Fixes #7349

### CODE REVIEW

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ByteCodeDumper.java [hunk 1]
```diff
@@ -171,25 +171,30 @@ private static IDATA j9bcutil_dumpBytecodes(PrintStream out, J9ROMClassPointer r
...
-		if (System.getenv().containsKey("j9bcutil_dumpBytecodes")) {
-			envVarDefined = true;
-			environment = System.getenv().get("j9bcutil_dumpBytecodes");
+		try {
+			String environment = System.getenv().get("j9bcutil_dumpBytecodes");
+
+			if (environment != null) {
+				dumpBytecodes = Integer.valueOf(environment);
+			}
+		} catch (NumberFormatException e) {
```
Comment (pshipton) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ByteCodeDumper.java]: Not sure silent failure is the best behavior. An exception is more explicitly shows the problematic setting.

Comment (keithc-ca) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ByteCodeDumper.java]: No longer suppresses `NumberFormatException`.

**Comment by gacholio** [comment]
jenkins test sanity xlinux jdk8

**Comment by pshipton** [comment]
@keithc-ca we can put this into the 0.17 release, can you please create a PR.

**Comment by keithc-ca** [comment]
> @keithc-ca we can put this into the 0.17 release, can you please create a PR.

See #7413.

**Pull request merged**